sci-libs/clblast/files/patches/level2_xtrsv

```diff
@@ -0,0 +1,11 @@
+--- CLBlast/src/routines/level2/xtrsv.cpp	2022-05-21 08:44:13.811816246 -0600
```

Comment:
No need to put patches in a patches/ subdir.

Could you move them into just `files/`, and use a `.patch` filename suffix?

Reply:
Testing those changes now.

Reply:
Done. Pushed new commit.